src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
```

Comment:
use destructurization
```suggestion
export function Person({ person: { age, partnerName, sex } }) {
```

Reply:
Not resolved.